runtime/src/migration.rs

```diff
@@ -117,9 +115,10 @@ impl<T: Trait> Module<T> {
 
             potential_id += <T as storage::data_object_storage_registry::Trait>::DataObjectStorageRelationshipId::one();
         }
-    }
 
-    fn clear_storage_data_directory_data() {
+        storage::data_object_storage_registry::NextRelationshipId::<T>::put(<T as storage::data_object_storage_registry::Trait>::DataObjectStorageRelationshipId::one());
```

Comment:
`one()` works because the constant `DEFAULT_FIRST_RELATIONSHIP_ID` defined in the module is equal to 1.
But maybe better to just`kill()` the value, next time it is read it will use the default storage value in this case configured as the constant:

```rust
pub FirstRelationshipId get(first_relationship_id) config(first_relationship_id):
            T::DataObjectStorageRelationshipId = T::DataObjectStorageRelationshipId::from(DEFAULT_FIRST_RELATIONSHIP_ID);
```

Reply:
Fixed.